src/components/LayerTree/README.md

```diff
@@ -13,7 +13,7 @@ import BasicMap from 'react-spatial/components/BasicMap';
 const baseBright = new MapboxLayer({
   name: 'Base - Bright',
   group: 'baseLayer',
-  url: `https://maps.geops.io/styles/base_bright_v2/style.json?key=${apiKey}`,
+  url: `https://maps.geops.io/styles/base_bright_v2_ch.sbb.netzkarte/style.json?key=${apiKey}`,
```

Comment:
I'm not sure that we want spezifiscche sbb work in there, maybe use travic_v2 ? like in mobility toolbox